### DISCUSSION TITLE
2024/10/27

### DISCUSSION THREAD

**Discussion by YanWenKun** [YanWenKun/ComfyUI-Windows-Portable#25] [Announcements]
|How to use:|如何使用|
| --- | --- |
|1. Download all `.7z.00X` files|1. 下载所有 `.7z.00X` 文件|
|2. Open the `.001` file and extract|2. 打开第一个文件（`.001`）并解压|
|3. Edit `run_nvidia_gpu.bat` to fit your GPU|3. 编辑 `run_nvidia_gpu.bat` 以适配你的 GPU|
|4. Run `run_nvidia_gpu.bat` to start ComfyUI|4. 运行 `run_nvidia_gpu.bat` 启动 ComfyUI|
|Note: You can install custom nodes via "ComfyUI-Manager"|提示：使用“管理器”可以安装自定义节点|

### Changes | 变动
* Upgrade to PyTorch 2.5 + CUDA 12.4
* 升级到 PyTorch 2.5 + CUDA 12.4

- Using [wheels for Comfy3D](https://github.com/YanWenKun/ComfyUI-Windows-Portable/releases/tag/v5.3-wheels) compiled by me. May have compatibility issues (or even better compatibility).
- 用了我自己编译的 Comfy3D 依赖项 [wheel 文件](https://github.com/YanWenKun/ComfyUI-Windows-Portable/releases/tag/v5.3-wheels)，可能会有兼容性问题（也可能兼容性更好），暂不作为正式版发布。如果反馈良好，那真是性能和兼容性全都要了。

* Add custom node: PuLID
* 添加自定义节点：PuLID

### Versions | 版本信息
* Python 3.12.7
* CUDA 12.4
* PyTorch 2.5.0
* xFormers 0.0.28.post2

<hr /><em>This discussion was created from the release <a href='https://github.com/YanWenKun/ComfyUI-Windows-Portable/releases/tag/v6.0'>2024/10/27</a>.</em>

**Comment by Falkonar** [comment]
Where to install your wheels for Comfy3D

**Reply by YanWenKun** [reply]
They are already installed in the `python_embeded`.
If you need manual installation, see the doc:
https://github.com/YanWenKun/ComfyUI-Windows-Portable/blob/main/docs/compiling-wheels-for-comfy3d.adoc
